src/virtio-net-carrier-slirp.cpp

```diff
@@ -197,12 +201,18 @@ virtio_net_carrier_slirp::virtio_net_carrier_slirp() {
     }
 
     // Setup host forward ports
-    struct in_addr host_addr {};
-    struct in_addr guest_addr {};
-    host_addr.s_addr = htonl(INADDR_ANY);
-    guest_addr.s_addr = htonl(SLIRP_DEFAULT_IPV4_VDHCP_START);
-    if (slirp_add_hostfwd(slirp, false, host_addr, 2022, guest_addr, 22) < 0) {
-        throw std::runtime_error("failed to forward host TCP port 2022 to guest port 22");
+    for (const auto &hostfwd : config.hostfwd) {
+        struct in_addr host_addr {};
+        struct in_addr guest_addr {};
+        // ??(edubart): Should we allow configuring host/guest addrs?
```

Comment:
I think that forcing host address to be INADDR_ANY is not ideal. I commented on the config already.

Reply:
Should I use `INADDR_LOOPBACK` instead? I am trying to avoid this to be configurable.

Reply:
This is now being done and the comment was removed.